ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-4.0__ocp-4-13.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: apollo-ci
     namespace: stackrox
-    tag: stackrox-test-0.3.58
+    tag: stackrox-test-0.3.59
```

Comment:
I think this should also be `0.3.57.1`. All the CI for a release branch should run in the same environment. This probably originates from a copy paste error. Similar to https://github.com/openshift/release/pull/39523.